### DISCUSSION TITLE
Using Aspire with a Blazor Web project that has a separate Client WASM project

### DISCUSSION THREAD

**Discussion by markharrison** [microsoft/aspire#4350] [Q&A]
Should I be able to use Aspire with a Blazor Web project that has a separate Client WASM project.
Im not clear how the AppHost orchestration can wire things up so the  Client WASM project can connect with the API service ?

![image](https://github.com/dotnet/aspire/assets/5177824/9cef8e8e-cce3-49ab-9e41-24a6a5e76978)

**Comment by jacobwi** [comment]
Anyone figured this one out?
